src/styles/blocks/catalog.scss

```diff
@@ -17,9 +18,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $duration-time, color $duration-time;
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      cursor: pointer;
```

Comment:
just set it for default class, not :hover pseudoclass. because pointer wil be changed only on mouse hover by default

Reply:
Okey, one minutes